var/spack/repos/builtin/packages/rose/package.py

```diff
@@ -46,20 +46,56 @@ class Rose(Package):
     depends_on("autoconf@2.69", type='build')
     depends_on("automake@1.14", type='build')
     depends_on("libtool@2.4", type='build')
-    depends_on("boost@1.54.0")
-    depends_on("jdk@8u25")
+    depends_on("boost@1.47.0:")
 
-    def install(self, spec, prefix):
-        # Bootstrap with autotools
+    variant('tests', default=False, description='enable the tests directory')
```

Comment:
What exactly does this variant do? If it installs more stuff, that's fine. But if it runs the unit tests during installation, you don't want a variant for this.